src/sb_utils.R

```diff
@@ -8,25 +24,100 @@ sb_replace_files <- function(sb_id, ..., file_hash){
   hashed_filenames <- c()
   if (!missing(file_hash)){
     hashed_filenames <- yaml.load_file(file_hash) %>% names %>% sort() %>% rev()
-    for (file in hashed_filenames){
-      item_replace_files(sb_id, files = file)
+    if(use_task_table) {
+      do_item_replace_tasks(sb_id, hashed_filenames, sb_upload_utils_src)
+    } else {
+      for (file in hashed_filenames){
+        item_replace_files(sb_id, files = file)
+      }
     }
   }
-  files <- c(...)
+  
   if (length(files) > 0){
-    item_replace_files(sb_id, files = files)
+    if(use_task_table) {
+      do_item_replace_tasks(sb_id, files, sb_upload_utils_src)
+    } else {
+      item_replace_files(sb_id, files = files)
+    }
   }
   
 }
 
-sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
+sb_render_post_xml <- function(sb_id, ..., xml_file = NULL, use_task_table = TRUE, sb_upload_utils_src = "sb_utils.R"){
```

Comment:
Looks like you don't have the dir to `"sb_utils.R"` here. That file doesn't exist, but `"src/sb_utils.R"` does

I'd suggest calling this arg `sources` instead of `sb_upload_utils_src`. It is shorter, and also more flexible (and aligns with the task table input argument).

Reply:
I didn't include the dir because that could be different depending on where this function is being used. I call the one for this repo in the README example.  I guess I should remove the default to make it actually robust for reuse.